LICENSE

```diff
@@ -1040,9 +1040,9 @@ The externally maintained libraries used by Node.js are:
 - zlib, located at deps/zlib, is licensed as follows:
   """
     zlib.h -- interface of the 'zlib' general purpose compression library
-    version 1.2.11, January 15th, 2017
+    version 1.2.13, October 13th, 2022
```

Comment:
The zlib on the main branch is still v1.2.12: https://github.com/nodejs/node/commit/b6c108f0926c2df734f0d3c32c9419bf4b2a0192

Reply:
`deps/zlib/zlib.h` says 1.2.13. 🤷 `deps/zlib/LICENSE` says 1.2.12 though. 🤔

Reply:
https://chromium.googlesource.com/chromium/src/third_party/zlib indicates that 8bbd6c31 is 1.2.13, so I think this commit is correct and the zlib LICENSE file didn't get updated for the release. So I think we're good.